lint/action.yaml

```diff
@@ -30,6 +30,10 @@ inputs:
   npm-token:
     required: false
     description: The Node Package Manager (npm) authentication token. This token is used to authenticate against the NPM registry.
+  internal-dep-prefix:
```

Comment:
I would call this `internal-dependency-prefixes` (no need to abbreviate to `dep` and use prefixes to indicate that it can take several entries)

Reply:
And lastly, if you could update the readme with this new input https://github.com/open-turo/actions-node/blob/main/lint/README.md